### PR TITLE
[hive_input][benchmark]add doc and hive_input

### PR DESCRIPTION
**Pull request by poson**
1. add benchmark doc
2. hive input and test

### CODE REVIEW

File: easy_rec/python/test/hive_input_test.py [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+    #   feature_dict, label_dict = sess.run([features, labels])
+    #   for key in feature_dict:
+    #     print(key, feature_dict[key][:5])
+    #
+    #   for key in label_dict:
+    #     print(key, label_dict[key][:5])
+    return 0
+
+  @RunAsSubprocess
+  def test_mmoe(self):
```
Comment (chengmengli06) [inline easy_rec/python/test/hive_input_test.py:224]: remove useless code

File: easy_rec/python/test/hive_input_test.py [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+class HiveInputTest(tf.test.TestCase):
+
+  def __init__(self, methodName='HiveInputTest'):
+    super(HiveInputTest, self).__init__(methodName=methodName)
+    self._input_path = 'census_income_train_simple'
+
+  @RunAsSubprocess
+  def test_hive_input(self):
+    data_config_str = """
```
Comment (chengmengli06) [inline easy_rec/python/test/hive_input_test.py:32]: long config maybe better to put into a config file

File: easy_rec/python/protos/dataset.proto [hunk 3]
```diff
@@ -245,4 +247,6 @@ message DatasetConfig {
         HardNegativeSampler hard_negative_sampler = 103;
         HardNegativeSamplerV2 hard_negative_sampler_v2 = 104;
     }
+
+    optional HiveConfig hive_config = 27;
 }
```
Comment (chengmengli06) [inline easy_rec/python/protos/dataset.proto:252]: it is better to be placed in pipeline.proto where KafkaInput is placed.

**Review by chengmengli06 — CHANGES_REQUESTED** [review]

File: easy_rec/python/protos/dataset.proto (hunk 3 above)
Comment (poson) [reply easy_rec/python/protos/dataset.proto:252]: move to pipeline.proto

File: easy_rec/python/test/hive_input_test.py (hunk 2 above)
Comment (poson) [reply easy_rec/python/test/hive_input_test.py:32]: it’s part of config file , test config style.

**Review by chengmengli06 — APPROVED** [review]

**Pull request merged**